### PR TITLE
Add metadata for openstack 2023.1 (antelope) charms

### PR DESCRIPTION
**Pull request by ajkavanagh**
This is for enabling recipes and other automated tooling.

### CODE REVIEW

File: charmed_openstack_info/data/lp-builder-config/openstack.yaml [hunk 1]
```diff
@@ -458,6 +494,97 @@ projects:
     charmhub: nova-compute-nvidia-vgpu
     launchpad: charm-nova-compute-nvidia-vgpu
     repository: https://opendev.org/openstack/charm-nova-compute-nvidia-vgpu.git
+    # TODO: charmhub migration is not completed yet for this charm.  It needs
+    # backporting to bionic (I think).
+    branches:
```
Comment (freyes) [inline charmed_openstack_info/data/lp-builder-config/openstack.yaml:499]: this charm was released during the yoga cycle, it has no support for releases older than yoga.

**Review by freyes — CHANGES_REQUESTED** [review]

File: charmed_openstack_info/data/lp-builder-config/openstack.yaml (hunk 1 above)
Comment (ajkavanagh) [reply charmed_openstack_info/data/lp-builder-config/openstack.yaml:499]: Maybe?  Judging by this commit https://opendev.org/openstack/charm-nova-compute-nvidia-vgpu/commit/8296660bb31eb5aa4f1be9bf7ad3a29678ff133a it looks like at least focal-ussuri should be supported?

Comment (ajkavanagh) [reply charmed_openstack_info/data/lp-builder-config/openstack.yaml:499]: And previously, bionic-queens was supported: https://opendev.org/openstack/charm-nova-compute-nvidia-vgpu/commit/f8c5b6ceff7786f4127aac21449d60db556f24cf

Comment (freyes) [reply charmed_openstack_info/data/lp-builder-config/openstack.yaml:499]: it has no branches though - https://github.com/openstack/charm-nova-compute-nvidia-vgpu/branches .

Comment (ajkavanagh) [reply charmed_openstack_info/data/lp-builder-config/openstack.yaml:499]: Oh, I agree; hence I left it as a todo.  I guess we need to work out which ones are really needed and then update this as needed.

**Comment by freyes** [comment]
this change needs to be rebase to pick [this change](https://github.com/openstack-charmers/charmed-openstack-info/pull/17) up.

**Comment by ajkavanagh** [comment]
> this change needs to be rebase to pick [this change](https://github.com/openstack-charmers/charmed-openstack-info/pull/17) up.

Ack, updated now; thanks.

**Pull request merged**